nexus/types/src/deployment/planning_input.rs

```diff
@@ -33,6 +33,28 @@ use std::net::IpAddr;
 use strum::IntoEnumIterator;
 use uuid::Uuid;
 
+/// Describes the current CockroachDB cluster settings we care about.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct CockroachdbSettings {
+    /// `version`
```

Comment:
Yeah, we should pick one. I'll go to "CockroachDB settings", I think that's fine.

In the full context of CockroachDB, "settings" could refer to cluster settings or node settings, the latter of which are set via the `cockroach start` command that we control in the SMF manifest. Within the context of Nexus, though, the only settings it can see are cluster settings.